spec/support/ems_refresh_helper.rb

```diff
@@ -40,6 +50,13 @@ def assert_inventory_not_changed(before, after)
           end
         end
       end
+
+      def with_vcr(suffix = nil, &block)
```

Comment:
Aside, I wonder if we should move away from custom VCR cassette names, and instead use the automatic VCR cassette names that are used via tagged specs.

Reply:
So the only issue with that is that it becomes difficult to e.g. reuse the full refresh vcr as the start for more specific targeted refresh tests.  There could be a nice way of dealing with that I just haven't looked into it much.